README.md

```diff
@@ -32,45 +34,66 @@ allprojects {
 }
 ```
 
-project build.gradle
+project **build.gradle**
 ```groovy
 dependencies {
     commonMainApi("dev.icerock.moko:permissions:0.1.0")
 }
 ```
 
-settings.gradle  
+**settings.gradle**
 ```groovy
 enableFeaturePreview("GRADLE_METADATA")
 ```
 
-## Usage
-TODO
+## List of supported permissions
 
+The full list can be found in `dev.icerock.moko.permissions.Permission` enum.
+
+* Camera: **Permission.CAMERA**
+* Gallery: **Permission.GALLERY**
+* Storage: **Permission.STORAGE**
+* Fine location: **Permission.LOCATION**
+* Coarse location: **Permission.COARSE_LOCATION**
+
+## Usage example
```

Comment:
please replace to `Usage` as in other moko libs readme

Reply:
Fixed.